### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by rdrpenguin04**
woo.

### CODE REVIEW

File: package-lock.json
Comment (kilbouri) [inline package-lock.json:1]: Can you summarize what packages got upgraded? 1167 lines of changes is um... not something I want to try and decipher myself....

Comment (rdrpenguin04) [reply package-lock.json:1]: I ran `npm up`

Comment (kilbouri) [reply package-lock.json:1]: And it should have spat out a list of changes. Which of the packages in `package.json` did it upgrade?

Comment (rdrpenguin04) [reply package-lock.json:1]: Many

**Comment by kilbouri** [comment]
Is there a reason behind this upgrade?

**Comment by rdrpenguin04** [comment]
Maintenance, keeping versions up to date. Ideally this'll just be automated at some point
